src/Sarif.UnitTests/Writers/SarifLoggerTests.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Diagnostics;
```

Comment:
FileVersionInfo lives in this namespace. Odd but true.

---
In reply to: [63250501](https://github.com/Microsoft/sarif-sdk/pull/271#discussion_r63250501) [](ancestors = 63250501)